### ISSUE TITLE
Powershell v7.2.0.preview10 exception on startup in PSReadline

### ISSUE THREAD

**Issue by hawko2600** [PowerShell/PSReadLine#2911]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PSReadLine)
- [X] Search the existing issues, especially the pinned issues.

### Exception report

```console
### Exception

System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Prediction.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics)
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken)
```
```


### Screenshot

N/A. This occurs when starting PowerShell, the state beforehand is "PowerShell is not running"

### Environment data

```console
### Environment
PSReadLine: 2.2.0-beta2
PowerShell: 7.2.0-preview.10
OS: Microsoft Windows 10.0.19043
BufferWidth: 120
BufferHeight: 9001

Last 0 Keys
```

### Steps to reproduce

1. Add registry key to enable preview PowerShell to install via Windows Update
2. Run Windows Update and install the preview version of PowerShell
3. Launch PowerShell preview from the Start Menu

### Expected behavior

PowerShell loads and works without triggering exceptions at every prompt

### Actual behavior

PowerShell loads and triggers this exception at every prompt.

**Comment by daxian-dbw** [comment]
This is a known issue and has been addressed in 2.2.0-beta3 release of PSReadLine. Please upgrade to use 2.2.0-beta3 with the latest 7.2.0-preview.9 PowerShell.

**Comment by hawko2600** [comment]
Thanks, confirmed fixed.
